src/context/api/useUserInfoQuery.tsx

```diff
@@ -0,0 +1,12 @@
+import { useQuery } from 'react-query';
+import { getUserInfoRequest } from './request/getUserInfoRequest';
+
+const useUserInfoQuery = () => {
+  const user = useQuery('user', () => getUserInfoRequest(), {
+    staleTime: Infinity,
+    retry: 0,
```

Comment:
I am not sure if those settings are required because probably `staleTime` and `retry` are default set to those values